pizza_cutter/des_pizza_cutter/_coadd_slices.py

```diff
@@ -586,6 +590,16 @@ def _coadd_slice_inputs(
             wcs_position_offset=wcs_position_offset,
         )
 
+    # fpacking will produce negative outputs on the way out so we truncate the
+    # masked fraction to zero if it is less than 1e-3 to help eliminate this
+    # we also do the same for values close to 1
+    msk = (np.isclose(interp_se_frac, 0, rtol=1e-3, atol=1e-3) & (interp_se_frac != 0))
+    if np.any(msk):
+        interp_se_frac[msk] = 0
+    msk = (np.isclose(interp_se_frac, 1, rtol=1e-3, atol=1e-3) & (interp_se_frac != 1))
+    if np.any(msk):
+        interp_se_frac[msk] = 1
```

Comment:
note fpack will not preserve ones; so we may need to clip again on reading if we want them no larger than 1, although I'm  not sure it matters much

Reply:
yup. we can likely remove this.